src/features/Chat/ChatMenu.tsx

```diff
@@ -28,7 +28,7 @@ const ChatMenu = () => {
         <Buttons>
```

Comment:
Not part of this PR it seems, but can you check on this file lines 24-26. (and also lines 93-95)

Couldnt we just instead of 3 evaluations use 

```javascript
t(`menu.title.${currentCategory}`)
```

Reply:
Also I think the ChatMenu-file is a bit big, maybe we could have a folder 
Menu and Windows, and there components-folders too. 


Suggestion how it would look like

- features
    - Chat
        - index.tsx
        - components
            - Menu
                - index.tsx
                - TopActions
                ...
            - Window
                - index.tsx
                - Message
                ...

Reply:
I'm pretty sure I tried that and it's not possible to evaluate the i18n keys like that. 🤔 

Yeah this is a good idea, but in the next PR perhaps?

Reply:
It's definitely possible.
Currently the ChatCategory-union has a different key for the chats-status. (active vs chats)

Reply:
About the structuring, if you are planning to improve the component composition in next PR, it ofcourse can be done in that one

Reply:
Ah I tried it again and it works fine, don't know why I thought it didn't! 😅

Thanks, nice one!